arch/arm/configs/tegra3_android_defconfig

```diff
@@ -1683,6 +1683,7 @@ CONFIG_CHARGER_SMB347=y
 # CONFIG_CHARGER_GPIO is not set
 CONFIG_BATTERY_BQ27541=y
 # CONFIG_TEGRA_BPC_MGMT is not set
+CONFIG_FORCE_FAST_CHARGE=y
```

Comment:
Hmm I doubt this config parameter is supported by the tegra kernel, but I'll investigate